src/renderer/screens/account/AccountHeaderActions.js

```diff
@@ -97,7 +134,27 @@ const AccountHeaderActions = ({ account, parentAccount, openModal, t }: Props) =
           ) : null}
 
           <ReceiveAction account={account} parentAccount={parentAccount} onClick={onReceive} />
-          {availableOnExchange ? <BuyActionDefault currency={currency} onClick={onBuy} /> : null}
+          {actions && actions.length > 0 && (
```

Comment:
I believe we agreed on not using the logical `&&` operator to return jsx conditionally because we fell into crashes that were a nightmare to locate (in the generated asar file) and use ternaries with a null fallback cc @gre

Reply:
Ho yes indeed when you have a single return statement it can do a React render error :+1: